src/features/force-update-modal/force-update-modal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button, Modal, Text, View } from 'react-native';
+
+import { translate } from 'localization/hooks';
+
+import { isUpdated, openStore } from 'network/services/force-update';
+
+import { styles } from './force-update-modal.styles';
+
+interface ForceUpdateProps {}
+
+const ForceUpdateModal = ({}: ForceUpdateProps) => (
+  <Modal animationType="slide" transparent={true} visible={isUpdated()}>
```

Comment:
Did you test it on an Android device? 
What would happen if an Android user hits the back button?

Reply:
I test it in android and the modal keeps showing after pressing the back button.